code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -195,7 +195,7 @@ GLOBAL_LIST_INIT(titaniumglass_recipes, list(
 	singular_name = "titanium glass sheet"
 	icon_state = "sheet-titaniumglass"
 	item_state = "sheet-titaniumglass"
-	materials = list(MAT_TITANIUM=MINERAL_MATERIAL_AMOUNT, MAT_GLASS=MINERAL_MATERIAL_AMOUNT)
+	materials = list(MAT_TITANIUM=MINERAL_MATERIAL_AMOUNT/2, MAT_GLASS=MINERAL_MATERIAL_AMOUNT)

```

Comment:
`* 0.5` is faster than `/ 2`

Reply:
And how many femtoseconds does it save for compilation?

Reply:
It's not for compiling that it saves times; it's running it. In BYOND, multiplication is faster than division, as @RemieRichards demontrated; it just makes more sense to utilize multiplication where it can be applied easily.

Reply:
My point is that it's most likely calculated by the compiler instead of on runtime since it's a static value, so there isn't a point in going out of the way to optimize something that already takes negligible amount of time once.

Reply:
Except byond's compiler is hot garbage.
We know it does a minor amount of constant folding, that's the only optimisation we actually know it does, and it doesn't even fully commit to that.